signature-aggregator/aggregator/aggregator_test.go

```diff
@@ -43,6 +44,7 @@ func instantiateAggregator(t *testing.T) (
 		1024,
 		sigAggMetrics,
 		messageCreator,
+		time.Now().Add(-1*time.Minute),
```

Comment:
What's the purpose of the -1 minute?

Reply:
This is the `etnaTime` parameter. This would make the aggregator in this test run using the post-etna code paths since in this case `Etna` was activated a minute ago

Reply:
3d7707dc370ad953a957f6297b9f03cfd27c9431